src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs

```diff
@@ -4,7 +4,11 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+#if EF7
+using Microsoft.EntityFrameworkCore;
+#else
 using System.Data.Entity;
+#endif
```

Comment:
I'm not quite sure what you mean, but EFCore does have some API changes which incompatible to EF6, so those projects exist, if we want to keep only 1 copy of Restier code, perhaps we must add some wrapper layer for EF.